pallets/ddc-clusters/src/lib.rs

```diff
@@ -116,6 +117,60 @@ pub mod pallet {
 		OptionQuery,
 	>;
 
+	#[pallet::genesis_config]
+	pub struct GenesisConfig<T: Config> {
```

Comment:
Might be a good idea to add:
```
ddc_clusters: Default::default()
```
Somewhere in 
https://github.com/Cerebellum-Network/blockchain-node/blob/911c21ea16f8f56b858e85a5ef5731684b369bdc/node/service/src/chain_spec.rs#L239

For consistency and explicitness even if default is implemented.

Reply:
Already added in the end because no specific ordering scheme implemented.

https://github.com/Cerebellum-Network/blockchain-node/pull/153/files#diff-292a131483a432bb2352be1af3f2421027cc368f9a1dc63ec5fc567ecc85b0c3R238